lib/rubocop/cop/rails/where_missing.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Check for the use `left_joins` and `where` method for finding missing relationship records.
+      #
+      # This cop is enabled in Rails 6.1 or higher.
+      #
+      # @example
+      #   # bad
+      #   Foo.left_joins(:foo).where(foos: { id: nil })
+      #
+      #   # good
+      #   Foo.where.missing(:foo)
```

Comment:
This seems to be easier to understand if it is concrete like the style guide example:
https://rails.rubystyle.guide/#finding-missing-relationship-records

```suggestion
      #   # bad
      #   Post.left_joins(:author).where(authors: { id: nil })
      #
      #   # good
      #   Post.where.missing(:author)
```

Reply:
Indeed it is. I updated this PR.